articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Understanding quantum computing
+description: What are quantum computers and how do they use the principles of quantum physics?
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.understanding
+---
+
+# Understanding quantum computing
+
+Quantum computing uses the principles of quantum physics to process information. Because of this, quantum computing requires a different approach than classical computing.  One example of this is the processor used in quantum computers.  Where classical computers use familiar silicon-based chips, quantum computers use quantum particles such atoms, ions, photons, or electrons.  
+
+These quantum particles behave according to the laws of quantum physics, leveraging concepts such as probabilistic computation, superposition, and entanglement, which provide the basis for quantum algorithms that harness the power of quantum computing to solve complex problems. This article describes some of the essential concepts of quantum physics on which quantum computing is based.
+
+## A bird’s-eye view of quantum physics
+Quantum physics, also called quantum mechanics or quantum theory, is a branch of physics that deals with particles at the atomic and subatomic levels. At the quantum level, however, many of the laws of physics you take for granted don’t apply. Superposition, quantum measurement, and entanglement are three phenomena that are central to quantum computing.  
+
+### Superposition vs. binary computing
+
+Imagine that you are exercising in your living room. You turn all the way to your left and then all the way to your right. Now turn to your left and your right at the same time. You can’t do it (not without splitting yourself in two, at least).  Obviously, you can’t be in both of those states at once – you can’t be facing left and facing right at the same time.
+
+However, if you were a quantum particle, then you could be *facing left* AND *facing right* AND *anywhere in-between* at the same time due to a phenomenon known as **superposition** (also known as **coherence**).
+
+A quantum particle such as an electron has its own “facing left or facing right” property called **spin**, which can be either up or down (or positive or negative for a photon), or to make it more relatable to classical binary computing, let’s just say 1 or 0. When a quantum particle is in a superposition state, it’s a linear combination of an infinite number of states between 1 and 0, but you don’t know which one it will be until you actually look at it, which brings up our next phenomenon, **quantum measurement**.
```

Comment:
> (or positive or negative for a photon)

Generally 𝑍-axis of the Poincaré sphere (the photonic equivalent of the Bloch sphere) is labeled by |↕⟩ and |↔⟩ rather than "positive" and "negative" or "up" and "down."

Reply:
I'm trying to keep it very basic here - is the up/down reference too misleading, or can it just be used to make the binary point?